### ISSUE TITLE
MDX parser can't parse inline object styles with multiple properties

### ISSUE THREAD

**Issue by kentcdodds** [prettier/prettier#7120]
**Prettier 1.19.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4AIDOMAngDZwC8wwAtgIYAeA6pujABZK4DMADDwDS469ABJxMAczYxOvHgF95APjaZUAeizYlIfiAgAHGJmj5koWgCdLEAO4AFKwjMpa2CC10gARpdpgAazgYAGUDf0woCWQYSwBXOD1pahJGVXh8cLA4EOdMY2x8omRwfDM9SPw4Sxh7Pwk6ZAAzWhIqvQArfHoAIT9A4JDaajgAGUi4Ztb2kC76EMiJMgBFOIh4KbbEkHDLKssS6nR6LwNLSJhmVjZkAA4BHZsqxj8DErO4fexJvQBHNfgdUMLhAtHwAFooHA4OgYV5LHB-pgEXVaA1aJsZlVqJgYvFtvhFisAZMkC0tnoYLRvFd2MgAEyUvyYEiLADCEGojRQUGgPxAcSqABVqS5yTNsAkAJJQWGwEJgc5GACCspCxDImLgiiAA)
```sh
--parser mdx
```

**Input:**
```mdx
<div style={{maxWidth: 300, maxHeight: 300}}>hi</div>
```

**Output:**
```mdx
SyntaxError: Unexpected closing tag "div". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags (1:48)
> 1 | <div style={{maxWidth: 300, maxHeight: 300}}>hi</div>
    |                                                ^
```

**Expected behavior:**

It parses without trouble.

**Comment by skovy** [comment]
I believe this was working in `1.18.2` (or at least doesn't error?).

I tested this input:

```mdx
<!-- test.mdx -->

## Heading

<div style={{ maxWidth: 300, maxHeight: 300 }}>hi</div>
```

With these commands:

**Latest (1.19.1)**

`npx prettier@latest --write test.mdx --parser mdx`

```
[error] test.mdx: SyntaxError: Unexpected closing tag "div". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags (1:50)
[error] > 1 | ## Heading
[error]     |                                         ^
[error]   2 |
[error]   3 | <div style={{ maxWidth: 300, maxHeight: 300 }}>hi</div>
[error]   4 |
```

**1.18.2**

`npx prettier@1.18.2 --write test.mdx --parser mdx`

```
npx: installed 1 in 1.116s
test.mdx 64ms
```

It looks like this may now be fixed by https://github.com/prettier/prettier/pull/6949 but unreleased?

**Comment by thorn0** [comment]
> It looks like this may now be fixed by #6949 but unreleased?

Exactly.
